requirements-dev.txt

```diff
@@ -1,19 +1,12 @@
 black
```

Comment:
If `black` is installed and run with pre-commit, maybe it doesn't need to also be here? Same for mypy?

Reply:
It's still useful to have in the dev environment. If you're running VS Code, then black is a requirement for "Format Document..." (although I'm not 100% sure this is still the case since they came out with the Black VS Code extension.)

Reply:
Also, ruff is a linter, not a type checker, so I think we want to keep mypy, at least for now.

Reply:
> Also, ruff is a linter, not a type checker, so I think we want to keep mypy, at least for now.

Yes, I was more thinking about mypy being on the slow side to use in a pre-commit workflow, not about eliminating it entirely. I don't want to touch mypy for this PR in any case, one thing at a time is enough for me  :laughing:

Reply:
>It's still useful to have in the dev environment. If you're running VS Code, then black is a requirement for "Format Document..." (although I'm not 100% sure this is still the case since they came out with the Black VS Code extension.)

Interesting. I'm not a VSCode user, but I assumed that would integrate with pre-commit as well. Makes sense!